typed-store/src/rocks/mod.rs

```diff
@@ -451,3 +425,16 @@ pub fn open_cf<P: AsRef<Path>>(
     };
     Ok(rocksdb)
 }
+
+/// TODO: Good description of why we're doing this
+#[inline(always)]
```

Comment:
Please replace with `#[inline]`, the always is not the best idea here, even though it sounds like it should:
http://troubles.md/rust-optimization/#dont-use-inline-always